subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -27,15 +28,21 @@ LOG_MODULE_DECLARE(fast_pair, CONFIG_BT_FAST_PAIR_LOG_LEVEL);
 
 BUILD_ASSERT(FP_ACCOUNT_KEY_LEN == FP_CRYPTO_AES128_KEY_LEN);
 
+enum wait_for {
+	WAIT_FOR_ACCOUNT_KEY_BIT_POS,
+	WAIT_FOR_PERSONALIZED_NAME_BIT_POS,
```

Comment:
btw. Since we use uint8_t to store bitmask, consider adding a build assert verifying that all of the bits fit in memory.